ethcore/src/error.rs

```diff
@@ -134,12 +134,12 @@ pub enum BlockError {
 
 /// Newtype for Display impl to show seconds
 #[derive(Debug, Clone, From, PartialEq, Eq)]
-pub struct OutOfBoundsTime(OutOfBounds<SystemTime>);
+pub struct OutOfBoundsTime(OutOfBounds<Duration>);
 
 impl fmt::Display for OutOfBoundsTime {
 	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
 		let seconds = self.0
-			.map(|st| st.elapsed().unwrap_or_default().as_secs());
+			.map(|st| st.as_secs());
```

Comment:
This is not the same anymore, because before it printed out the time elapsed from that the `SystemTime` was created

The change just prints out it the duration as secs